### PR TITLE
omf doctor

### PR DESCRIPTION
**Pull request by bpinto**
Requires: https://github.com/oh-my-fish/oh-my-fish/pull/90

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,9 +87,9 @@ Add a new package. To add a theme use `omf submit` _`themes/<name>`_ _`<url>`_.
 
 Make sure to [send us a PR][omf-pulls-link] to update the registry.
 
-#### `omf query` _`<variable name>`_
+#### `omf doctor`
 
-Use to inspect all session variables. Useful to  dump _path_ variables like `$fish_function_path`, `$fish_complete_path`, `$PATH`, etc.
+Use to troubleshoot before opening an [issue][omf-issues-new].
```
Comment (bpinto) [inline README.md]: @bobthecow what do you think?

Comment (bpinto) [inline README.md]: @chgu82837 Could you help on the translation here?

Comment (bobthecow) [inline README.md]: I'd link all of "opening an issue"

**Comment by derekstavis** [comment]
Haha, I see what you did here!

```
⋊> ~/.l/s/omf on doctor ◦ omf doctor                                                                                                          21:54:30
Please note that these warnings are just used to help the Oh My Fish maintainers
with debugging if you file an issue. If everything you use Oh My Fish for is
working fine: please don't worry and just ignore them. Thanks!
⋊> ~/.l/s/omf on doctor ◦ brew doctor                                                                                                         21:54:36
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry and just ignore them. Thanks!
```

**Comment by bpinto** [comment]
:sweat_smile:

File: pkg/omf/cli/omf.doctor.fish [hunk 2]
```diff
@@ -0,0 +1,26 @@
+function __omf.doctor.theme
+  if test -e ~/.config/fish/functions/fish_prompt.fish
+    set -l failed
```
Comment (derekstavis) [inline pkg/omf/cli/omf.doctor.fish]: not needed (look below)

File: pkg/omf/cli/omf.doctor.fish [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+    echo
+    echo (omf::err)"Warning: "(omf::off)"`fish_prompt.fish` is overridden."
+    echo "`fish_config` command persists the prompt to ~/.config/fish/functions/fish_prompt.fish."
+    echo "That file takes precedence over Oh My Fish theme's prompt. To use Oh My Fish's theme, run:"
+    echo "`rm ~/.config/fish/functions/fish_prompt.fish`"
```
Comment (derekstavis) [inline pkg/omf/cli/omf.doctor.fish]: `return 1`

File: pkg/omf/cli/omf.doctor.fish [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  end
+
+  set -q failed
```
Comment (derekstavis) [inline pkg/omf/cli/omf.doctor.fish]: `return 0`

**Comment by derekstavis** [comment]
Some :lipstick: could be

```fish
function __omf.doctor.theme
  if test -e ~/.config/fish/functions/fish_prompt.fish
    echo
    echo (omf::err)"Warning: "(omf::off)(omf::em)"fish_prompt.fish"(omf::off)" is overridden."
    echo (omf::em)"fish_config"(omf::off)" command persists the prompt to "(omf::em)"~/.config/fish/functions/fish_prompt.fish"(omf::off)
    echo "That file takes precedence over Oh My Fish theme's prompt. To use Oh My Fish's theme, run:"

    set_color green
    echo "rm ~/.config/fish/functions/fish_prompt.fish"

    return 1
  end

  return 0
end

function omf.doctor
  set_color -o
  echo "Please note that these warnings are just used to help the Oh My Fish maintainers
with debugging if you file an issue. If everything you use Oh My Fish for is
working fine: please don't worry and just ignore them. Thanks!"

  __omf.doctor.theme; or set -l doctor_failed

  if not set -q doctor_failed
    echo
    echo (omf::em)"Your system is ready to swim."(omf::off)
  end
end

```

**Comment by derekstavis** [comment]
Awesome! :+1:

**Pull request merged**